toolkit/scss/forms/_summary.scss

```diff
@@ -269,13 +269,13 @@ table.summary-item-body tbody .summary-item-field-first-narrow-no-padding {
 
 .summary-item-field-headings-visible {
```

Comment:
I'm curious as to where we actually use this class (plenty of usage of the one below). I wonder if we can delete it altogether?

Reply:
This is the class that gets applied to the `<thead>` itself.

Hmm.... but yes, I do suspect that `border-top` has no effect on `<thead>`. Or at least it *shouldn't*. Who knows about weird-browser-support.